dask_jobqueue/core.py

```diff
@@ -548,5 +548,10 @@ def _job_id_from_submit_output(self, out):
 
         return job_id
 
-    def worker_key(self, worker_state):
+    @staticmethod
+    def worker_key(worker_state):
```

Comment:
The method wasn't easy to serialize because the class has some unserializable elements.  Fortunately we don't need `self` here, so I changed this to a `staticmethod` in order to remove need to serialize the class.